tasks/quality_control/advanced_metrics/task_busco.wdl

```diff
@@ -33,19 +34,25 @@ task busco {
     if [ -f ~{samplename}/short_summary.specific.*.~{samplename}.txt ]; then
 
       # grab the database version and format it according to BUSCO recommendations
-      cat ~{samplename}/short_summary.specific.*.~{samplename}.txt | grep "dataset is:" | cut -d' ' -f 6,9 | sed 's/,//' | sed 's/ / (/' | sed 's/$/)/' | tee DATABASE
+      # pull line out of final specific summary file
+      # cut out the database name and date it was created
+      # sed is to remove extra comma and to add parentheses around the date and remove all tabs
+      # finally write to a file called DATABASE
+      cat ~{samplename}/short_summary.specific.*.~{samplename}.txt | grep "dataset is:" | cut -d' ' -f 6,9 | sed 's/,//; s/ / (/; s/$/)/; s|[\t]||g' | tee DATABASE
       
-      # extract the results string
-      cat ~{samplename}/short_summary.specific.*.~{samplename}.txt | grep "C:" | tee BUSCO_RESULTS
+      # extract the results string; strip off all tab and space characters; write to a file called BUSCO_RESULTS
+      cat ~{samplename}/short_summary.specific.*.~{samplename}.txt | grep "C:" | sed 's|[\t]||g; s| ||g' | tee BUSCO_RESULTS
```

Comment:
Thanks for adding this for a cleaner output!

Reply:
No problem, but FYI it was a necessary change since the text in BUSCO output file `~{samplename}/short_summary.specific.*.~{samplename}.txt` had changed slightly after the version upgrade to 5.7.1. They added a bunch of tabs to make it look "pretty" to read to the human eye but less parse-able

I first attempted simply upgrading the BUSCO docker image and the results string output was wonky and filled with tabs and/or spaces so this cleans up the output to make it an identical format to how we were parsing with the previous version of BUSCO